src/payment/strategies/moneris/moneris.ts

```diff
@@ -1,9 +1,43 @@
+/**
+ * A set of stringified CSS to apply to Moneris' IFrame fields.
+ * CSS attributes should be converted to string.
+ * Please note that ClassNames are not supported.
+ *
+ * IE:
+ * ```js
+ * {
+ *      cssBody: 'background:white;';
+ *      cssTextbox: 'border-width:2px;';
+ *      cssTextboxPan: 'width:140px;';
+ *      cssTextboxExpiry: 'width:40px;';
+ *      csstexboxCvd: 'width:40px;';
+ * }
+ * ```
+ *
+ * When using several attributes use semicolon to separate each one.
+ * IE: 'background:white;width:40px;'
+ */
 export default interface MonerisStylingProps {
+    /**
+     * Stringified CSS to apply to the body of the IFrame.
+     */
     cssBody?: string;
+    /**
+     * Stringified CSS to apply to each of input fields.
```

Comment:
Not sure if this is a convention we've already set, but why `Textbox` rather than `Input`?

Also, why prefix everything with `css` when it's already in a `style` namespace?

Reply:
This is the way that Moneris is naming these fields, these are the fields that we are passing to Moneris to build the iframe